### ISSUE TITLE
Lossless Audio?

### ISSUE THREAD

**Issue by samgurung** [Alex313031/apple-music-desktop#6]
Does this play lossless? if so how do we set it?

**Comment by shuiiiiiii** [comment]
> Does this play lossless? if so how do we set it?

No, Its a wrapper of the Web version of the Apple Music which does not let you stream Lossless quality rather at 256kbps max.
